tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.robot

```diff
@@ -0,0 +1,166 @@
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume create --name=${containerVolName} --opt VolumeStore=nfsVolumeStoreDefault
+    Should Be Equal As Integers  ${rc}  1
+    Should Contain  ${output}  Error response from daemon: A volume named ${containerVolName} already exists. Choose a different volume name.
+
+
+*** Test Cases ***
+Simple docker volume create
+    Pull image  ${busybox}
+
+    ${rc}  ${outputDefault}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume create --opt VolumeStore=nfsVolumeStoreDefault
+    Should Be Equal As Integers  ${rc}  0
+
+    Set Suite Variable  ${UnnamedNFSVolContainer}  unnamednfsVolContainer
+    Set Suite Variable  ${nfsDefaultVolume}  ${outputDefault}
+
+    Verify NFS Volume Basic Setup  ${nfsDefaultVolume}  ${UnnamedNFSVolContainer}  ${NFS_DEFAULT_IP}  rw
+
+Docker volume create named volume
+    ${rc}  ${outputDefault}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume create --name nfs_default_%{VCH-NAME} --opt VolumeStore=nfsVolumeStoreDefault
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Equal As Strings  ${outputDefault}  nfs_default_%{VCH-NAME}
+
+    Set Suite Variable  ${namedNFSVolContainer}  namednfsVolContainer
+    Set Suite Variable  ${nfsNamedVolume}  ${outputDefault}
+
+    Verify NFS Volume Basic Setup  ${nfsNamedVolume}  ${namedNFSVolContainer}  ${NFS_DEFAULT_IP}  rw
+
+Docker volume create already named volume
+    Verify NFS Volume Already Created  ${nfsDefaultVolume}
+
+    Verify NFS Volume Already Created  ${nfsNamedVolume}
+
+Docker volume create with possibly invalid name
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume create --name=test??? --opt VolumeStore=nfsVolumeStoreDefault
+    Should Be Equal As Integers  ${rc}  1
+    Should Be Equal As Strings  ${output}  Error response from daemon: volume name "test???" includes invalid characters, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed
+
+Docker single write to file from one container in named volume
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --name ${createFileContainer} -d -v ${nfsNamedVolume}:/mydata ${busybox} /bin/top -d 600
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} exec -i ${createFileContainer} sh -c "echo 'The Texas and Chile flag look similar.\n' > /mydata/test_nfs_file.txt"
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} exec -i ${createFileContainer} sh -c "ls mydata/"
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  test_nfs_file.txt
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} exec -i ${createFileContainer} sh -c "cat mydata/test_nfs_file.txt"
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  The Texas and Chile flag look similar.
```

Comment:
@mhagen-vmware, I was able to do the `run` command w/o naming a container after reading your comment, but I remember why I wrote this part of the test that way.  I wanted to make sure that I was dealing with just one named running container in the background.  The exec calls in this test are just for that one container that has already been mounted with an NFS volume.  Also, I use this running container later on in this test for another scenario.